apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Utils.cs

```diff
@@ -19,6 +19,28 @@ namespace Google.Cloud.Diagnostics.Common.Tests
 {
     internal class Utils
     {
+        private const string _projectEnvironmentVariable = "TEST_PROJECT";
+
+        /// <returns>The test project Id.</returns>
```

Comment:
This should have a summary tag since that what InelliSense will display in the IDE (same comment below).